server/src/bw.js

```diff
@@ -271,7 +271,7 @@ async function getBedgraph(req, res, file, pa) {
 	/* read and plot all bedgraph lines from a locus, without summary
 	pa={minv,maxv,bedgraphdir}
 	 */
-	if (pa.minv == undefined || pa.maxv == undefined) throw 'Y axis scale must be defined for bedgraph track'
+	if (pa.fixminv == undefined || pa.fixmaxv == undefined) throw 'Y axis scale must be defined for bedgraph track'
```

Comment:
should this be a strict equality, `pa.fixminv === undefined`, so that `0` is allowed?